hicommonwealth/commonwealth
# Remove Tokens model from database.

## ~~DO NOT MERGE UNTIL #6099 IS MERGED~~

## Link to Issue
Closes: #6134 

## Description of Changes
- Removes Token model incl migration to drop table.

## Test Plan
- Verify up/down of migration.
- Verify server starts properly post-migration. No routes affected.
